### ISSUE TITLE
Griefprevention softmute not working

### ISSUE THREAD

**Issue by smuddgge** [HelpChat/ChatChat#211]
```yaml
ChatChat: #65
Server Version: 1.19.4
Grief Prevention: 16.18.1
```

I had a look though the code but I could not find why the new grief prevention code is not working.

Same as before, when you softmute a player, players are still able to see what they are saying.

**Comment by BlitzOffline** [comment]
You have to go to plugins/ChatChat/extensions.yml and enable the griefprevention mute extension. Let me know if it still isn't working after that.

**Comment by smuddgge** [comment]
```yaml
addons:
  deluxechat:
    inverse_priorities: true
    unicode_permission:
      public_chat: true
      private_chat: true
  towny:
    channels: false
  discordsrv:
    channels_bridging: false
  essentials:
    vanish: false
  supervanish:
    vanish: true
    soft_mute: false
  griefprevention:
    soft_mute: true
```

We have softmute set to true but still doesn't work.

![Screenshot 2023-06-21 at 10 04 57 pm](https://github.com/HelpChat/ChatChat/assets/68658429/d4f1d964-d26a-4daf-90f2-e771fce87ccc)
![IMG-20230621-WA00032](https://github.com/HelpChat/ChatChat/assets/68658429/3f7da0a2-e5e8-48bf-bf4d-d3527e668816)

**Comment by BlitzOffline** [comment]
Just making sure but have you restarted the server after toggling it to true? Any changes in extensions.yml require a restart to apply.

**Comment by smuddgge** [comment]
I think we have found the issue, when you reload the plugin the config regenerates like this :

```yaml
addons:
  deluxechat:
    inverse_priorities: false
    unicode_permission:
      public_chat: true
      private_chat: true
  towny:
    channels: false
  discordsrv:
    channels_bridging: false
  essentials:
    vanish: false
  supervanish:
    vanish: false
    soft_mute: true
 ```
 
 Maybe you could look though the code, may be related to why softmute is not working

**Comment by smuddgge** [comment]
![Screenshot 2023-06-22 at 10 22 38 am](https://github.com/HelpChat/ChatChat/assets/68658429/036f94a5-40e7-4979-9f1c-c07e132b9fd9)
Ahh found it!

**Comment by smuddgge** [comment]
Made fix here #212

**Comment by BlitzOffline** [comment]
Ah. Good catch. Tested everything but right before finishing that PR I've made some changes that I didn't test and I guess somehow I managed to forget to change something when copy pasting the changes.
